cpp/src/slice2java/Gen.cpp

```diff
@@ -4314,268 +4314,92 @@ Slice::Gen::ProxyVisitor::visitInterfaceDefEnd(const InterfaceDefPtr& p)
     out << eb;
 
     out << sp;
-    writeDocComment(
```

Comment:
I deleted these doc-comments because they're redundant. If you override a method, and don't supply a doc-comment, you automatically inherit one in Java.

I double-checked, and these comments are identical to the ones on `ObjectPrx` (the base), except in very small ways (like `@returns A proxy` vs `@returns A new proxy`)